packages/cli/contracts/src/codegen/tables/Dynamics1.sol

```diff
@@ -32,9 +36,7 @@ struct Dynamics1Data {
 library Dynamics1 {
   /** Get the table values' field layout */
   function getFieldLayout() internal pure returns (FieldLayout) {
-    uint256[] memory _fieldLayout = new uint256[](0);
-
-    return FieldLayoutLib.encode(_fieldLayout, 5);
+    return _fieldLayout;
   }
```

Comment:
do we even need `getFieldLayout` if we have it as a constant above? wonder if it's worth just exporting that as a named constant like we do for table ID